### DISCUSSION TITLE
Are failed  uploads resumable / what happens to temporary files

### DISCUSSION THREAD

**Discussion by hhanzo1** [Forceu/Gokapi#161] [Q&A] [answered]
I was just wondering how does Gokapi manage failed uploads? Can the upload be resumed?

Are incomplete uploads deleted off the disk after a certain time to reclaim disk space?

Thanks.

**Comment by Forceu** [comment, marked as answer]
Ah the moment it is not possible to resume uploads (unless you are using the API) and all temporary files are deleted after 24 hours

**Comment by hhanzo1** [comment]
OK Sounds good. Thank you for the quick reply!
